dspace-api/src/main/java/org/dspace/content/dao/impl/MetadataValueDAOImpl.java

```diff
@@ -70,7 +72,13 @@ public void deleteByMetadataField(Context context, MetadataField metadataField)
     public MetadataValue getMinimum(Context context, int metadataFieldId)
             throws SQLException
     {
-        String queryString = "SELECT m FROM MetadataValue m JOIN FETCH m.metadataField WHERE m.metadataField.id = :metadata_field_id ORDER BY text_value";
+	String queryString;
+	if(DatabaseUtils.getDbType().equals(DatabaseUtils.DBMS_ORACLE)){
+	    queryString = "SELECT m FROM MetadataValue m JOIN FETCH m.metadataField WHERE m.metadataField.id = :metadata_field_id ORDER BY DBMS_LOB.substr(text_value , 4000 , 1)";
+	}else{
```

Comment:
Treat this as a workaround then. 
An hibernate expert could remake the query using criterias. As a reference: http://stackoverflow.com/questions/29744868/how-to-sort-by-clob-column-with-hibernate-db2